ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -238,7 +259,7 @@ class TabBarItemView: UIControl {
         if let window = window {
             let primaryColor = Colors.primary(for: window)
```

Comment:
You might want to grab tokens.aliasTokens.foreground[.brandRest] instead of Colors.primary(for: window), though I think it's more likely that this should just be a tokenized value?

Reply:
For some reason I forgot about globalTokens.brandColors[.primary], but that's a better option.

Reply:
First off, yeah this should definitely be a token as well (`selectedColor` perhaps?). 

Second, agreed that we should use alias tokens rather than global tokens whenever possible.